src/main/java/sirius/kernel/commons/Strings.java

```diff
@@ -575,17 +575,19 @@ public static String cleanup(@Nullable String inputString, @Nonnull Iterable<Una
         return value;
     }
 
-    protected static final String REGEX_DETECT_XML = "</?[a-zA-Z][^>]*>";
+    protected static final String REGEX_DETECT_XML_TAGS = "</?[a-zA-Z][^>]*>";
+    protected static final String REGEX_DETECT_ENTITIES = "&[a-zA-Z]+[0-9]{0,2};";
```

Comment:
I'm not sure you cover all kinds of escape chars here:
`&#38;`, `&#x00F7;` are some samples of it...

Reply:
Good point :)